sim/paladin/items.go

```diff
@@ -16,6 +16,10 @@ const (
 	LibramDiscardedTenetsOfTheSilverHand = 209574
 	LibramOfBenediction                  = 215435
 	LibramOfDraconicDestruction          = 221457
+	HerosBrand							 = 231328
+	zandalarFreethinkersBreastplate      = 231329
+	zandalarFreethinkersBelt             = 231330
+	grileksCharmOfValor                  = 231285
```

Comment:
🔨 Small knit. Would you mind ordering these by ID descending and then capitalizing the first letter of the variable names to be consistent with others?